src/unix/kqueue.c

```diff
@@ -34,6 +34,10 @@
 #include <fcntl.h>
 #include <time.h>
 
+#ifndef EV_OOBAND
+#define EV_OOBAND  EV_FLAG1
```

Comment:
The issue is Mac OS X

You will find the same define in BOOST:
http://www.boost.org/doc/libs/1_61_0/boost/asio/detail/kqueue_reactor.hpp

And in the `event.h` file from Apple themselves:
https://opensource.apple.com/source/xnu/xnu-2050.7.9/bsd/sys/event.h

I can't find when support was added.